packages/features/bookings/lib/handleNewBooking/test/dynamic-group-booking.test.ts

```diff
@@ -112,7 +122,6 @@ describe("handleNewBooking", () => {
             responses: {
               email: booker.email,
               name: booker.name,
-              location: { optionValue: "", value: "New York" },
```

Comment:
Since the event has Cal Video enabled as default, this test is now attempting to supply a location different from what is set in the event. This allows them to hijack the location to whatever they choose. The right behavior is to reject such requests.